android/app/build.gradle.kts

```
@@ -201,14 +201,31 @@ android {
+            val envKeyPassword = System.getenv("JABOOK_KEY_PASSWORD")
+
+            val resolvedStoreFile = keystoreProperties.getProperty("storeFile") ?: envStoreFile
+            val resolvedStorePassword = keystoreProperties.getProperty("storePassword") ?: envStorePassword
+            val resolvedKeyAlias = keystoreProperties.getProperty("keyAlias") ?: envKeyAlias
+            val resolvedKeyPassword = keystoreProperties.getProperty("keyPassword") ?: envKeyPassword
+
+            if (
+                !resolvedStoreFile.isNullOrBlank() &&
+                !resolvedStorePassword.isNullOrBlank() &&
+                !resolvedKeyAlias.isNullOrBlank() &&
+                !resolvedKeyPassword.isNullOrBlank()
```

detekt (1.23.8)

[warning] This condition is too complex (4). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)